### ISSUE TITLE
Vuex plugins support

### ISSUE THREAD

**Issue by Lop001** [mrcrowl/vuex-typex#1]
It would be great if plugins could be inserted into StoreBuilder.vuexStore()
```
export interface StoreBuilder<R> {
    /** Get a ModuleBuilder for the namespace provided */
    module<S>(namespace: string, state: S): ModuleBuilder<S, R>;
    /** Output a Vuex Store after all modules have been built */  
    vuexStore(plugins: Array<(store:Store<R>) => void>): Store<R>;
}
```

**Comment by mrcrowl** [comment]
Good idea.  Included in v1.0.6.  

Instead of a `plugins` argument, I've gone with an optional `overrideOptions`, which is a partial StoreOptions<R>.  i.e.
```
export interface StoreBuilder<R> {
    /** Get a ModuleBuilder for the namespace provided */
    module<S>(namespace: string, state: S): ModuleBuilder<S, R>;

    /** Output a Vuex Store after all modules have been built */
    vuexStore(): Store<R>

     /** Output a Vuex Store and provide options, e.g. plugins -- these take precedence over any auto-generated options */
    vuexStore(overrideOptions?: StoreOptions<R>): Store<R>;
}
```

**Comment by Lop001** [comment]
It works like a charm. Thank you!
